documentation/src/main/asciidoc/stories/assembly_server_getting_started.adoc

```diff
@@ -8,15 +8,29 @@ Install the server distribution, create a user, and start your first {brandname}
 ifdef::community[]
 image:https://infinispan.org/assets/images/documentation/icon-get-started.png[Get started icon,height=32]
 link:https://infinispan.org/get-started/[Start running {brandname} in 4 easy steps]
+endif::community[]
 
 [discrete]
 == Ansible collection
 
-link:https://github.com/ansible-middleware/infinispan[Ansible collection for {brandname}]
-
 Automate installation of {brandname} clusters with our Ansible collection that optionally includes Keycloak caches and cross-site replication configuration.
 The Ansible collection also lets you inject {brandname} caches into the static configuration for each server instance during installation.
+
+//Community content
+ifdef::community[]
+[source,options="nowrap",subs=attributes+]
+----
+ansible-galaxy collection install middleware_automation.infinispan
+----
 endif::community[]
+//Downstream content
+ifdef::downstream[]
+[source,options="nowrap",subs=attributes+]
+----
+ansible-galaxy collection install redhat.infinispan
```

Comment:
instructions on how to run this? 
```
$ ansible-galaxy collection install redhat.infinispan
Starting galaxy collection install process
Process install dependency map
ERROR! Failed to resolve the requested dependencies map. Could not satisfy the following requirements:
* redhat.infinispan:* (direct request)
```

Reply:
Hmm. Looks like the command has changed. I'm not too sure now. 

Can you access it here? https://console.redhat.com/ansible/automation-hub/repo/published/redhat/jboss_data_grid/

I think maybe we should just point to this link or just the repo itself.

Reply:
Thanks, I've pushed some changes. I think we should just link the repo and point users to the automation hub instead of adding too much 3rd party documentation.